### PR TITLE
#10 add transitions using ENTER and add autocursore on input

### PR DESCRIPTION
**Pull request by Alkhimovmv**
1. Cursor already activated in the field for answer inputting
2. Each transition happen using enter
3. Refactor

### CODE REVIEW

**Review by AlexKay28 — CHANGES_REQUESTED** [review]

File: static/js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+document.onkeyup = function (e) {
+    e = e || window.event;
```
[the remark this answers is not in the document]
Comment (Alkhimovmv) [reply static/js/scripts.js:2]: This variable can be called e, evt or event, this is a common practice. 
I think that in this case you need to call it evt. But e is used to shorten the code

File: templates/practice_ask.html [hunk 2]
```diff
@@ -14,6 +14,6 @@ <h5>main page ask</h5>
 {% block body %}
   <h2>Translate: {{first_language}}</h2>
   <form action="/practice" method="post">
-    <input type="string" name="second_language_answer", id="second_language_answer">
+    <input type="string" name="second_language_answer", id="second_language_answer" autofocus>
```
[the remark this answers is not in the document]
Comment (Alkhimovmv) [reply templates/practice_ask.html:17]: Ok, before merging, I will pull these changes

[the remark this answers is not in the document]
Comment (AlexKay28) [reply templates/practice_ask.html:17]: kk

File: static/js/scripts.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlexKay28) [reply static/js/scripts.js:2]: ok, just worried
pls, always try to call variables explicitly. 
It increases code readability and it helps other developers (not experienced in JS) to understand better at least)

**Review by AlexKay28 — APPROVED** [review]

**Pull request merged**